main.js

```diff
@@ -39,13 +39,17 @@ import { generateRounds } from './lib/rounds.js'
     const reader = new FileReader()
 
     reader.onload = (ev) => {
-      const parties = parseCsv(ev.target.result)
+      let parties = parseCsv(ev.target.result)
 
-      if (!validate(parties.length)) {
+      if (!validate(parties.length) || !parties.every(p => 'name' in p && 'continent' in p)) {
```

Comment:
Add another validation to ensure that uploaded file contains `name` and `continent` property